integration-tests/update-local-embrace.sh

```diff
@@ -40,14 +58,17 @@ if [ "$test_harness_only" = false ]; then
   ./pack.sh ../packages/react-native-tracer-provider/ artifacts/embrace-io-react-native-tracer-provider-local.tgz
 fi
 
-# build and pack the test harness
-npm --prefix test-harness run build
-./pack.sh  test-harness/ artifacts/embrace-io-react-native-test-harness-local.tgz
-
-# update all test apps with the sdk packages and the test harness
-# TODO EMBR-4923 put test apps within a common folder and use lerna to run this command across all of them
-npm --prefix basic-test-app add \
-  ./artifacts/embrace-io-react-native-local.tgz \
-  ./artifacts/embrace-io-react-native-tracer-provider-local.tgz \
-  ./artifacts/opentelemetry-instrumentation-react-native-navigation-0.1.0.tgz \
-  ./artifacts/embrace-io-react-native-test-harness-local.tgz
\ No newline at end of file
+if [ "$skip_test_harness" = false ]; then
+  # build and pack the test harness
+  npm --prefix test-harness run build
+  ./pack.sh  test-harness/ artifacts/embrace-io-react-native-test-harness-local.tgz
+fi
+
+if [ "$skip_sdk_packages" = false ] || [  "$skip_test_harness" = false ]; then
+  # update the test app with the sdk packages and the test harness
+  npm --prefix $test_app add \
+    ./artifacts/embrace-io-react-native-local.tgz \
+    ./artifacts/embrace-io-react-native-tracer-provider-local.tgz \
+    ./artifacts/opentelemetry-instrumentation-react-native-navigation-0.1.0.tgz \
```

Comment:
I don't see `opentelemetry-instrumentation-react-native-navigation-0.1.0` packed under the if block of `skip_sdk_packages`, is this meant to be packed manually? (if so, it makes sense because it's coming from the other repo by now, but just trying to confirm)

Reply:
my bad! `opentelemetry-instrumentation-react-native-navigation-0.1.0.tgz` is actually committed in `integration-tests/artifacts`. ignore this comment

Reply:
added a quick note about it to the script